dao/lecture_halls.go

```diff
@@ -45,6 +45,10 @@ func GetLectureHallByID(id uint) (model.LectureHall, error) {
 	return lectureHall, err
 }
 
+func DeleteLectureHall(id uint) error {
+	return DB.Delete(&model.LectureHall{}, id).Error
```

Comment:
What happens with streams that are scheduled for the lecture hall we're about to delete? IMO, streams.lecture_hall_id should be set to `null` here. e.g. using DB.Raw. Also, camera presets for the lecture hall could be deleted

Reply:
true! thanks for the catch! 👍

Reply:
@joschahenningsen I think this should be fine now, used `DB.Exec` instead of `DB.Raw` :)